### PR TITLE
Adding 050 job and hooks and updates with new/latest images

### PR DESCRIPTION
**Pull request by matzew**
/cc @mgencur

### CODE REVIEW

File: ci-operator/config/openshift/knative-eventing-sources/openshift-knative-eventing-sources-release-v0.5.0.yaml [hunk 1]
```diff
@@ -0,0 +1,135 @@
+tag_specification:
```
Comment (matzew) [inline ci-operator/config/openshift/knative-eventing-sources/openshift-knative-eventing-sources-release-v0.5.0.yaml]: file generated with the "generator" script, we added yesterday, @mgencur

File: ci-operator/config/openshift/knative-eventing-sources/openshift-knative-eventing-sources-release-v0.5.0.yaml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    tag: base
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+canonical_go_repository: github.com/knative/eventing-sources
+binary_build_commands: make install
+test_binary_build_commands: make test-install
+tests:
+- as: e2e
+  commands: "INTERNAL_REGISTRY=image-registry.openshift-image-registry.svc:5000 ENABLE_ADMISSION_WEBHOOKS=false make test-e2e"
```
Comment (matzew) [inline ci-operator/config/openshift/knative-eventing-sources/openshift-knative-eventing-sources-release-v0.5.0.yaml]: this is something I see in serving and eventing (core) - doing the same for sources.

It is `  commands: "make test-e2e"` on files for older releases, @mgencur

Comment (mgencur) [reply ci-operator/config/openshift/knative-eventing-sources/openshift-knative-eventing-sources-release-v0.5.0.yaml]: The parameters are not required for eventing-sources. `make test-e2e` is enough.

**Pull request merged**